common/src/lib.rs

```diff
@@ -160,12 +160,15 @@ mod test {
 
     #[test]
     fn test_bootstrap_config_from_cli_and_load_configuration() {
-        // Cleanup old test files and folder if exist
-        if std::path::Path::new(&dir_utils::default_subdir("")).exists() {
-            std::fs::remove_dir_all(&dir_utils::default_subdir("")).unwrap();
-        }
+        use std::time::SystemTime;
+        let time = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .unwrap()
+            .as_micros();
+        let tmp_path = format!(".{}_temp_tari", time);
```

Comment:
There's also a TempDir crate and `string` function used in other tests to create temp directories.

Reply:
Nice! Used those.